todo_app/app.py

```diff
@@ -1,20 +1,18 @@
 from flask import Flask, render_template, request, redirect, url_for
-from todo_app.data.trello_items import get_items_all, add_item, close_item
+from todo_app.data.todo_app_items import get_items_all, add_item, close_item
```

Comment:
The fact this is such a small change in this file is a good sign that you picked a sensible interface for these functions, so that the trello-specific details were nicely encapsulated and now the Mongo ones are too!